### ISSUE TITLE
feature req: Upload markdown with images to Wiki from R

### ISSUE THREAD

**Issue by rubenarslan** [CenterForOpenScience/osfr#65]
This would be useful for my [codebook](https://github.com/rubenarslan/codebook) package.

**Comment by bgrich** [comment]
Hi @rubenarslan! Updating the wiki using osfr sounds like a great feature. I don't think we will be able to fit it into version 1 of the package, but it is definitely something that would be great for later versions.

**Comment by januz** [comment]
It would be great if you would add the possibility to manipulate a project's wiki. Is this still being considered?

**Comment by aaronwolen** [comment]
Yes, although I haven't figure out exactly how to expose the functionality in the package. What kind of interactions did you have in mind?

**Comment by januz** [comment]
Personally I'm interested in being able to generate and change all information (Github READMEs, documentation through `pkgdown`, OSF wiki) related to a research compendium from one place (R(Studio)) as part of a consistent workflow. Currently, I have to actively remind myself to change OSF wiki information when I push a new version of the compendium to Github and have to manually add the changes. If one could manipulate the OSF wiki through `osfr`, I could just add this step as part of the publication/versioning routine and be sure that it is done. Additionally, it would allow to re-use text templates in the different places (README, documentation, wiki, ...).
